addons/binding/org.openhab.binding.tellstick/README.md

```diff
@@ -1,104 +1,169 @@
-| timestamp       | DateTime     | This channel reports the last time this device state changed. |
+<table>
+<tr><td colspan="3">Actuators ([dimmer]/[switch]) support the following channels:</td></tr>
+<tr><td><b>Channel Type ID</b></td> <td><b>Item Type</b></td> <td><b>Description</b></td> </tr>
+<tr><td>dimmer</td><td>Number</td><td>This channel indicates the current dim level</td></tr>
+<tr><td>state</td><td>Switch</td><td>This channel indicates whether a device is turned on or off.</td></tr>
+<tr><td>timestamp</td><td>DateTime</td><td>This channel reports the last time this device state changed.</td></tr>
+</table>
+
+<table>
+<tr><td colspan="3">Sensors ([sensor]) support the following channels:</td></tr>
+<tr><td><b>Channel Type ID</b></td> <td><b>Item Type</b></td> <td><b>Description</b></td> </tr>
+<tr><td>humidity</td><td>Number</td><td>This channel reports the current humidity in percentage.</td></tr>
+<tr><td>temperature</td><td>Number</td><td>This channel reports the current temperature in celsius.</td></tr>
+<tr><td>timestamp</td><td>DateTime</td><td> This channel reports the last time this sensor was updates.</td></tr>
+</table>
+
+### Switchbased sensor workaround
+
+All switchbased sensors are binary and the goal is to represent them as a `contact` item in openhab. Eg. a door is open or closed and can't be altered by sending a radio signal. To achive that we will create a proxy item which is updated by a rule.
+
 
-Sensors support the following channels:
-| Channel Type ID | Item Type    | Description  |
-|-----------------|--------------|----------------------------------------------- |
-| humidity        | Number       | This channel reports the current humidity in percentage |
-| temperature      | Number       | This channel reports the current temperature in celsius |
-| timestamp       | DateTime     | This channel reports the last time this sensor was updates. |
+First create another proxy item for every sensor:
+
+```
+Switch front_door_sensor	"Front door"  <door>  {channel="tellstick:switch:1:7:state"}
+Contact front_door_proxy	"Front door"  <door>
+```
+
+Then create a rule which updates the proxy item:
+
+```
+rule "proxy_front_door_on"
+when
+	Item front_door_sensor changed to ON
+then
+	postUpdate(front_door_proxy, OPEN);		
+end
+
+rule "proxy_front_door_off"
+when
+	Item front_door_sensor changed to OFF
+then
+	postUpdate(front_door_proxy, CLOSED);		
+end
+```
 
 ## Full Example
 
-```xtend
-Bridge tellstick:telldus-core:1 "Tellstick Duo" []
+### tellstick.things
+
+```
+Bridge tellstick:telldus-core:1 "Tellstick Duo" [resendInterval="200"]
 Bridge tellstick:telldus-live:2 "Tellstick ZWave" [refresh="10000", publicKey="XXXXXXXX", privateKey="YYYYYY", token= "ZZZZZZZZ", tokenSecret="UUUUUUUUUU"]
 ```
+
+Devices are preferable discovered automatically.
+Add them either with karaf: `inbox approve <thingId>` or in paperUI. The bridges can also be added with PaperUI.
+### tellstick.items
```

Comment:
Damn it. Must be hard for you to always babysit all contributing developers. Apologies from me :+1: